include/emp/datastructs/map_utils.hpp

```diff
@@ -41,6 +41,19 @@ namespace emp {
     return in_map.find(key) != in_map.end();
   }
 
+  /// Take a map where the value is an integer and a key.
+  /// Increment value associated with that key if its present
+  /// or if its not add it and set it to 1
+  template <class MAP_T, class KEY_T>
+  inline void IncrementCounter( MAP_T & in_map, const KEY_T & key ) {
+    static_assert( std::is_same< typename MAP_T::key_type, int >::value);
+    if (emp::Has(in_map, key)) {
```

Comment:
The goal is for this to be as generic as possible. Also, since  it's trying to  be a counter (kind of like Counter in Python), the default value is 1 not 0

Reply:
Makes sense!